tomviz/EmdFormat.cxx

```diff
@@ -552,6 +582,23 @@ bool EmdFormat::read(const std::string& fileName, vtkImageData* image)
     spacing[0] = static_cast<double>(dim3[1] - dim3[0]);
     image->SetSpacing(spacing);
   }
+  std::string units = "[n_m]"; // default to nanometers
+  if (d->attribute(emdNode + "/dim1", "units", units)) {
+    if (units == "[deg]") {
+      QVector<double> angles;
+      for (unsigned i = 0; i < dim1.size(); ++i) {
+        angles.push_back(dim1[i]);
+      }
+      DataSource::setTiltAngles(image, angles);
+    } else if (units == "[rad]") {
+      QVector<double> angles;
+      for (unsigned i = 0; i < dim1.size(); ++i) {
+        // Convert radians to degrees since tomviz assumes degrees everywhere.
+        angles.push_back(dim1[i] * 180.0 / M_PI);
```

Comment:
The use of M_PI is causing a build failure on Windows.

Reply:
I've switched to `vtkMath::Pi()`